content/en/internal/configuration-database/users-roles/managing-accounts/_index.md

```diff
@@ -0,0 +1,154 @@
+---
+title: "Understanding and manipulating user accounts"
+weight: 30
+description: >-
+     Understanding and manipulating user accounts.
+---
+
+## Authentication
+
+The GOCDB UI attempts to authenticate you in one of two ways (the REST style API
+applies x509 only):
+
+- First, by requesting an IGTF accredited user certificate from your browser. If
+a suitable certificate is detected, you will be asked to confirm selection of your
+certificate in your browser.
```

Comment:
```suggestion
certificate in your browser. **Note**: if a client certificate has been provided to GOCDB, it will take precedence over any IdP based authentication. 
```

Reply:
I've incorporated this into my PR for this file